Ports/patchutils/package.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='patchutils'
+version='0.4.2'
+useconfigure='true'
+files=(
+    "https://github.com/twaugh/patchutils/archive/refs/tags/${version}.tar.gz 2ff95f11946558ce63f4d1167abaccbffd49750152346d5304e03ad884304ad6"
```

Comment:
It seems like there are preconfigured tarballs available at `http://cyberelk.net/tim/data/patchutils/stable/`.